### DISCUSSION TITLE
Password recovery token -> PIN

### DISCUSSION THREAD

**Discussion by lid3rs** [hagopj13/node-express-boilerplate#239] [Ideas]
Hello, I'm utilising this boilerplate for react-native backend server. When user recovers a password, server sends a link with a token. I would like to send a PIN (4 digits combination) to user's email to verify it. I see this boilerplate has a lot of nicely written code + tests. If (is) there a way to implement easily PIN feature without breaking the huge chunk of code or writing new huge chunk of code? Or, maybe, there is somewhere ready to be taken PIN functionality repo that can be taken and copy-pasted here without reinventing the weel?

**Comment by amadebusuyi** [comment]
@lid3rs Do you still need this information? I can help you out.
